lib/worker/metrics-job.js

```diff
@@ -7,4 +7,9 @@ module.exports = async (job) => {
   syncStatusCounts.forEach((row) => {
     statsd.gauge('syncs', row.count, { status: row.syncStatus })
   })
+
+  const agedSyncCounts = await Subscription.agedSyncCounts()
```

Comment:
should we `Promise.all()` these two `async/await` functions so that we're not synchronously waiting for a then b?

Reply:
Can you give me a hand, what sort of code are you thinking @tebriel? These queries are fast, so I'm really going for the easiest to grok here.

Reply:
Yeah! I was thinking we could issue these SQL queries concurrently using [`Promise.all()`](https://developer.mozilla.org/en-US/docs/web/javascript/reference/global_objects/promise/all)
(I didn't test this, I just wrote it in the comment box, so there may be a syntax error)

```js
Promise.all([Subscription.syncStatusCounts(), Subscription.agedSyncCounts()]).then((values) => {
  values[0].forEach((row) => {
    statsd.gauge('syncs', row.count, { status: row.syncStatus })
  })
  values[1].forEach((row) => {
    statsd.gauge('aged_syncs', row.count, { status: row.syncStatus })
  })
})
```

Reply:
If those queries are bound to be fast, then I think it's fine to do them sequentially, just thinking we could parallelize sql queries and lean on Postgres to save some time/work.

Reply:
Ahh so the `values` is an array of the return values from each of the promises. Neat!

I can make that change in a followup PR. Great suggestion